apps/condo/domains/banking/components/BankAccountInfo.tsx

```diff
@@ -47,24 +47,20 @@ export const BankingInfo: React.FC<IBankAccountInfo> = ({ bankAccount }) => {
     }, [breakpoints])
 
     return (
-        <Col span={6} xs={24}>
-            <RequisitesContainer style={containerStyle}>
-                <Tag
-                    bgColor={ colors.gray[7]}
-                    textColor={colors.white}
-                >
-                    {accrualsAndPayments}
-                </Tag>
-                <Row style={{ marginBottom: '12px', marginTop: '12px' }}>
-                    <Col span={6}>
-                        <Typography.Title level={3}>{get(bankAccount, 'organization.name', '-')}</Typography.Title>
-                    </Col>
-                </Row>
+        <RequisitesContainer style={containerStyle}>
+            <Tag
+                bgColor={ colors.gray[7]}
+                textColor={colors.white}
+            >
+                {accrualsAndPayments}
+            </Tag>
+            <Row style={spacingStyle}>
+                <Typography.Title level={3}>{get(bankAccount, 'organization.name', '-')}</Typography.Title>
+            </Row>
 
-                {renderRow(TinMessage, get(bankAccount, 'tin', '-'))}
-                {renderRow(BicMessage, get(bankAccount, 'routingNumber', '-'))}
-                {renderRow(NumberMessage, get(bankAccount, 'number', '-'))}
-            </RequisitesContainer>
-        </Col>
+            {renderRow(TinMessage, get(bankAccount, 'tin', '-'))}
+            {renderRow(BicMessage, get(bankAccount, 'routingNumber', '-'))}
+            {renderRow(NumberMessage, get(bankAccount, 'number', '-'))}
```

Comment:
I think you need wrap it in `<Row gutter={[0,20]}> {...} </>` instead use margins from `spacingStyle` const